conf/log4j.properties

```diff
@@ -181,6 +181,7 @@ log4j.appender.FUSE_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%
 log4j.logger.io.grpc.netty.NettyServerHandler=OFF
 
 # Disable noisy INFO logs from ratis
+log4j.logger.org.apache.ratis.grpc.server.GrpcLogAppender=ERROR
```

Comment:
Are there useful ERROR log in this class beside the annoying one?

Reply:
This line will still allow log4j to show ERROR level logs for this class. I am sure there are useful WARN and INFO level logs, we can re-enable them per use case when we need to debug certain issues.

Reply:
that's dangerous.... emmm...